### PR TITLE
Fixes #7157 - Combo box style in reports

### PR DESCRIPTION
**Pull request by boaz0**
This is how it looks like now:
http://imgur.com/k8MgyWp.png

### CODE REVIEW

**Comment by mmoll** [comment]
[test]

**Comment by boaz0** [comment]
@isratrade - I think [that](http://i.imgur.com/pma9efN.png) is more like it.

File: app/helpers/reports_helper.rb [hunk 1]
```diff
@@ -39,10 +39,10 @@ def report_tag level
 
    def logs_show
     return unless @report.logs.size > 0
-    form_tag @report, :id => 'level_filter', :method => :get do
-      content_tag(:span, _("Show log messages:") + ' ') +
+    form_tag @report, :id => 'level_filter', :method => :get, :class => "form form-inline" do
+      content_tag(:span, _("Show:") + ' ') +
```
Comment (isratrade) [inline app/helpers/reports_helper.rb]: I suggest to get kyle baker's opinion of these UI issues. I'm not sure we need to create an inline form. I suggest to change this PR just to use bootstrap 3 classes and create a separate issue if you want to change the label text from "Show log messages:" to "Show" and remove the line break.

**Comment by boaz0** [comment]
@isratrade - your review, please. :smile:

**Comment by isratrade** [comment]
merged as ec4108a587d993abbf9f27181628e2a06f868a2d. Thanks @ripcurld00d !

**Pull request closed without merging**